tools/setup/lang.json

```diff
@@ -49,10 +49,12 @@
     "scala": 21,
     "scheme": 14,
     "sql": 32,
+    "spoiler": 50,
     "swift": 41,
     "tex": 40,
     "text": 1,
     "vb.net": 45,
     "vbnet": 45,
-    "xml": 1
+    "xml": 1,
+    "zpoiler": 50
```

Comment:
We probably don't need the second `zpoiler` name, given my suggested rename of the code.